docker/prod/nodejs/Dockerfile

```diff
@@ -26,4 +26,4 @@ FROM nginx:1.13-alpine
 ARG NODE_ENV
 COPY docker/prod/nodejs/nginx_${NODE_ENV}.conf /etc/nginx/conf.d/default.conf
 COPY --from=node /code /code
-COPY /ssl /code/ssl
+# COPY /ssl /code/ssl
```

Comment:
this is the prod docker file is this change relevant to deployment change?

Reply:
no. I'll create a different Dockerfile. But, I'm not sure where it should go. Should I create a different env folder ?

Reply:
Should I create a different env folder here ? like /vm/nodejs/Dockerfile ?